### DISCUSSION TITLE
Buttons become unavailable after pressing FC during the roast, looking for guidance

### DISCUSSION THREAD

**Discussion by ghost** [artisan-roaster-scope/artisan#1599] [Q&A]
Good evening everyone!

I am hoping the 3rd time posting this is the charm. For some reason GitHub keeps suspending my account after I post this request for help. This has happened to me twice already and I have no idea what I am doing to get suspended. Unfortunately GitHub support hasn't responded to any of my support tickets to get help with this. To post this message I setup a new account and am hoping it doesn't get suspended this time. I don't use GitHub for anything else other than participating in this discussion group so I can't imagine what I am doing to cause the suspension. Oh well... onto my question about Artisan. 

I am new to Artisan and have read the setup documentation. The problem I am having is that after pressing the FC button during the roast the rest of the buttons become unavailable after a period of time. I am not longer able to use the FC End button or Drop button. 

After playing around with this I am wondering if it has something to do with my phase temps, and that is kicking off an auto activation of the drop. As you will notice in the video I post the drop tag is added to the graph even though I never press the FC End or Drop buttons. 

Key moments in the video 
- .03 FC button pushed
- 1:07 Buttons grey out and Drop tag added to the graph

https://youtu.be/V-w44HcgJLs

I have separate video segment of using the Start and Charge buttons (in that order) so if anyone needs to see that I can share those also. My roasts are long so I thought I would focus on the issue I am seeing. 

Here is my settings file for reference also. 

[artisan-settings.txt](https://github.com/user-attachments/files/16005333/artisan-settings.txt)

Appreciate any feedback that can be offered. Crossing my fingers this account doesn't get suspended. 

Thanks

**Comment by poundy** [comment]
still have no idea on github accounts. 

You have autodrop detection turned on. Note the DROP marker on the graph and at that point, there are no other roast milestones left, ie roasting is over, so all the other buttons get greyed out.

**Reply by ghost** [reply]
Thanks for the reply, that is what it looked like but I wasn't seeing that in the configuration. I will look again. 

What would be causing the auto drop to be detected, just curious? Is that temp based? My setup is very basic, it's a glass drum roaster using a Mastech dual port meter, no automation of any kind. 

And yeah, the account suspension is crazy, I wish they would just tell me what I am doing to cause the suspension. It makes no sense to me what I would be saying or doing to trigger it. It's been really frustrating.

**Reply by Terracotta-6** [reply]
Disable autoDrop 
![73fbf72233f9091969b10c2a87ab5d0](https://github.com/artisan-roaster-scope/artisan/assets/149906183/f214b265-70e3-46f3-8579-fe2abb711f8d)

**Reply by ghost** [reply]
Yeah I see that now. Thanks!

**Reply by poundy** [reply]
absolutely autodrop was set. I reviewed your settings file. As Terracotta pointed out, that's where you unset it in the interface. 
Yes it is temperature based. That is why it sometimes does not work for people - if you have noisy thermocouples for example, sometimes the noise level can cause it to be incorrectly registered. It's trying to detect the obvious decrease in temps you'd see when you drop the coffee into the cooling tray (when using a drum roaster, typically) and ordinarily the decrease is otherwise an unwanted thing in normal roasting so must mark the drop point.

**Reply by ghost** [reply]
Appreciate the feedback. 

It makes sense now why that was happening. I am using glass drum roaster over a propane stove and heat swings have been an issue for me unless I am really careful. Since it doesn’t have a cover the heat dissipates quickly. I also blow in air during the roast to get the chaff out of the drum which gives me a much cleaner roast. That doesn’t help with temp swings. 

Overall I haven’t been able to get some really good roast with this set up and it’s one hobby my wife has told me I can’t give up 😁

I have this auto drop setting disabled now so I should be ready to go for the next roast. 

Thanks again.

**Reply by poundy** [reply]
if you can find a way to reintroduce heat or warmed air in with your airflow, you'll improve your results for sure. And yeah sounds like the insulation aspect really will be the stickler - to get the best results you really need to stabilise the temperatures. The generally accepted guiding principle is a gradually declining rate-of-rise, which is hard to do when you've got challenges in getting heat into the system and also not losing too much. But it comes to taste - the better you can get the taste, the less pressure from SWMBO ;)

**Reply by ghost** [reply]
Yeah, I have been thinking of a couple ways I could do that, just need to spend some more time on it. 

Eventually I will upgrade to the Kaleido M2 or something like that... the budget committee is going to take a lot of convincing!

**Reply by poundy** [reply]
budget committee needs to do without coffee for a little bit, just to prove how worth it the upgrade might be :)

**Reply by Terracotta-6** [reply]
Kaleido M2 selects cheap and noisy non-armored type K thermocouples and replaces them with low noise thermocouples.

**Reply by ghost** [reply]
That is good to know. Before I do buy anything I will be asking for advice here and searching past discussions. 

I have also looked at a Kaffa model and a Kaldi model.
